### ISSUE TITLE
says 1.10.2 has missing dependency.

### ISSUE THREAD

**Issue by tjthetinygamer** [BluSunrize/ImmersiveEngineering#2291]
so when I try and load this up on the newest 1.10.2 it says "you are missing a dependency!"

tried with every 1.10 version and it ALL does it.

**Comment by BluSunrize** [comment]
Could you provide a screenshot?
And a modlist?

**Comment by tjthetinygamer** [comment]
![error](https://user-images.githubusercontent.com/30061849/28064421-d9e0f2f8-65f1-11e7-8e75-2f3756cca5d4.png)

![modspart2](https://user-images.githubusercontent.com/30061849/28064722-fae36c6e-65f2-11e7-9c32-3d37dac1d37b.png)
![modspart3](https://user-images.githubusercontent.com/30061849/28064723-fae6979a-65f2-11e7-81a5-f9c87ab688bd.png)
![modspart1](https://user-images.githubusercontent.com/30061849/28064724-fae7e690-65f2-11e7-9c99-4d80daa054fd.png)



as requested.

**Comment by BluSunrize** [comment]
Update OneProbe.

**Comment by tjthetinygamer** [comment]
so i tried that. then it told me that it was missing stuff which at that point i went "I have waila I don't care anymore" and removed OneProbe and it runs just fine now. why that was it I don't understand because updating it didn't fix it it just broke OneProbe then too. None the less it loads now and I can play in this epic landscape I found and make the railroad mining town look exactly the way I wanted because your mod is the thing we needed to do it. Thank you.
